From 690bf48c956cbcc3770cf5d78e16e87511fd4023 Mon Sep 17 00:00:00 2001
From: Nicolas Geoffray <ngeoffray@google.com>
Date: Mon, 13 Sep 2021 17:28:00 +0100
Subject: [PATCH 01/13] [master] Update .oat version after disabling partial
 LSE.

This will ensure we don't take any .oat file that has the bogus
generated code.

Test: test.py
Bug: 197981962
Change-Id: Ic14d18d310bdcd408c1f6e2777ef53a041fb2f12
---
 runtime/oat.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/runtime/oat.h b/runtime/oat.h
index ab45b84888..6f5908587b 100644
--- a/runtime/oat.h
+++ b/runtime/oat.h
@@ -32,8 +32,8 @@ class InstructionSetFeatures;
 class PACKED(4) OatHeader {
  public:
   static constexpr std::array<uint8_t, 4> kOatMagic { { 'o', 'a', 't', '\n' } };
-  // Last oat version changed reason: Apex versions in key/value store.
-  static constexpr std::array<uint8_t, 4> kOatVersion { { '1', '9', '5', '\0' } };
+  // Last oat version changed reason: Disable partial LSE b/197818595.
+  static constexpr std::array<uint8_t, 4> kOatVersion { { '1', '9', '8', '\0' } };
 
   static constexpr const char* kDex2OatCmdLineKey = "dex2oat-cmdline";
   static constexpr const char* kDebuggableKey = "debuggable";
-- 
2.34.0

